src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import ErrorBoundary from '../../components/ErrorBoundary';
+import { connect } from 'unistore/react';
+import { GlobalState } from 'store';
+import actions, { StateApi } from 'actions';
+import DashboardDevice from 'components/Dashboard/DashboardDevice';
+import { CompositeFeature, DeviceState } from 'types';
+
+type OwnProps = RouteComponentProps;
+
+type Props = Pick<GlobalState, 'devices' | 'deviceStates'> & OwnProps & StateApi;
+
+const Dashboard: React.FC<Props> = (props) => {
+    return (
+        <ErrorBoundary {...props}>
```

Comment:
don't use ErrorBoundary boundary here, we have one on application level

Reply:
that doesn't make sense, this are props from Router,  same for u 
`const devicePageWithRouter = withRouter(DevicePage);` this is useless as Router is passing same props to your route, so pass props to `ConnectedDevicePage` instead of ErrorBoundary as this component is not doing anything with route props